packages/web3-providers/src/explorer/index.ts

```diff
@@ -0,0 +1,35 @@
+import urlcat from 'urlcat'
+import { toTransaction } from './helpers'
+import type { Transaction } from './types'
+
+export async function getLatestTransactions(
+    account: string,
+    url: string,
+    {
+        offset = 10,
+        apikey,
+    }: Partial<{
+        offset?: number
+        apikey?: string
+    }> = {},
+) {
+    const response = await fetch(
+        urlcat(url, {
+            module: 'account',
+            action: 'txlist',
+            address: account.toLowerCase(),
+            startBlock: 0,
```

Comment:
Better to use a larger number, e.g. the block number of tx in watched transactions or the original tx or something.

Reply:
The primary purpose of this API is to fetch the latest n transactions. The explorer API will handle this even with these extremely bad parameters.